innative/utility.cpp

```diff
@@ -136,7 +136,7 @@ namespace innative {
       return h;
     }
 
-    uint8_t GetInstruction(StringSpan ref)
+    uint16_t GetInstruction(StringSpan ref)
```

Comment:
Once again, if this is supposed to capture all instruction bytes in a single type, this type should be defined based off of the maximum number of instruction bytes.